### ISSUE TITLE
Graph of changes

### ISSUE THREAD

**Issue by javiavid** [DynamicPlayerSector/timethings#2]
I like the new plugin you have created, I have long notes that I modify many times and keeping track seems interesting to me.

I think that being able to have that control over time would be something very useful. Maybe with something similar to the github contributions graph or another type of graph.
Perhaps you could generate an external file in another folder that saves the data of when it was modified and for how long, this way the YAML would not be filled with too much data.

**Comment by DynamicPlayerSector** [comment]
Good idea! I was thinking of a similar thing, but I thought about implementing it using a sqlite database. It would keep a record of whenever you edited your files, with some cool down period, and then I could just show it in a nice UI since it should be no problem to analyze lots of values from a database.

I'm thinking on getting to work on this.

**Comment by javiavid** [comment]
Great!
